landing_page/spec/models/user_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+    let (:user){User.new}
+
+    context "don't create a user account with unacceptable credentials" do
```

Comment:
The context should represent a "state", a "pre-condition", not a behaviour, which would fall to the "it" blocks. Thus, I suggest the following:

```suggestion
    context "user account with unacceptable data" do
```

Reply:
commit a8b0890076739c912f85351d691d88a8f5d1ca6a